<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistWidget</class>
 <widget class="QWidget" name="PlaylistWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>258</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="animated">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>29</number>
     </attribute>
     <column>
      <property name="text">
       <string>#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Channel</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelSearch">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCategory">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="categoryBox"/>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="searchEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons/images/stop.png</normaloff>:/icons/images/stop.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>searchEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
